matchbox_socket/src/webrtc_socket/wasm.rs

```diff
@@ -311,9 +333,9 @@ async fn handshake_offer(
                 debug!("channel ready");
                 break;
             }
-            msg = signal_receiver.next() => {
+            msg = from_peer_rx.next() => {
```

Comment:
I think I'd prefer to keep `signal` in the name, so this is a consistent term through the project that makes it easy to distinguish from packets from peers etc.? Makes it easier to see why this is passed around to handhake, ice trickle etc.?

`signal_rx` or `peer_signal_rx`, `signals_from_peer_rx`?